interfaces/Portalicious/src/app/services/auth.service.ts

```diff
@@ -113,12 +87,16 @@ export class AuthService {
   }
 
   public async login(
-    credentials: { username: string; password?: string },
+    credentials?: { username: string; password?: string },
```

Comment:
```suggestion
    credentials: { username: string; password?: string },
```

Same as auth-strategy interface